tests/integration/dynamic_ip_test.py

```diff
@@ -70,7 +70,10 @@
 IPV6_DEFAULT_GATEWAY = "::/0"
 IPV4_DEFAULT_GATEWAY = "0.0.0.0/0"
 
+LEASE_FILE = "/tmp/nmstate.dnsmasq.lease"
+
 DNSMASQ_CONF_STR = """
+dhcp-leasefile={lease_file}
```

Comment:
Why is this needed? I think you could remove it. Thanks!

Reply:
The default lease file path might change. Better make it explicitly.

Reply:
The lease files location is defined here. What is not needed?

Reply:
But you are not using the variable `dhcp-leasefile` so.. maybe I am missing something..

Reply:
`dhcp-leasefile` is not a variable, its configuration of dnsmasq.

Reply:
@ffmancera , ping.